### PR TITLE
T23 - Voting Contest

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,40 @@
...
+
+      /// <summary>Determine the winner by which contestant has more votes</summary>
+      /// <param name="contestants">string</param>
+      /// <returns>It returns winner as char and voting as int data type</returns>
+      static (char, int) VotingContest (string contestants) {
+         List<(int, char)> voteCount = new ();
+         foreach (char c in contestants) {
+            if (char.IsLetter (c)) {
+               int count = contestants.Count (f => f == c);
+               (int, char) list = (count, c);
```
Comment (asfar-nazar) [inline Program.cs:31]: Why is this variable called a list ? Try to use appropriate names for variables

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,40 @@
...
+               voteCount.Add (list);
+            }
+         }
+         return (voteCount.MaxBy (x => x.Item1).Item2, voteCount.MaxBy (x => x.Item1).Item1);
```
Comment (asfar-nazar) [inline Program.cs:35]: Try to use tuples with named elements

File: Program.cs [hunk 3]
```diff
@@ -24,15 +24,14 @@ internal class Program {
...
          foreach (char c in contestants) {
             if (char.IsLetter (c)) {
                int count = contestants.Count (f => f == c);
-               (int, char) list = (count, c);
-               voteCount.Add (list);
+               voteCount.Add ((count, c));
             }
          }
-         return (voteCount.MaxBy (x => x.Item1).Item2, voteCount.MaxBy (x => x.Item1).Item1);
+         return (voteCount.MaxBy (x => x.Score).Candidate, voteCount.MaxBy (x => x.Score).Score);
```
Comment (asfar-nazar) [inline Program.cs:34]:
```suggestion
         var winner = voteCount.MaxBy (x => x.Score);
         return (winner.Candidate, winner.Score);
```

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 3 above)
Comment (Priyanka15-12) [reply Program.cs]: Changed

**Review by asfar-nazar — APPROVED** [review]

**Pull request closed without merging**